README.md

```diff
@@ -1,2 +1,166 @@
+[withRasterImages({
+	exclude:/images\/.*$/,
+})],
+
+```
+Setting the `url-loader` limit:
+```js
+// Set higher limit
+[withRasterImages({
+	options: {
+		limit: 300000,
+	},
+})],
+```
+
+Using limit and exclude/include to delineate between data URL items and standard items:
+```js
+// Exclude data-url directory
+[withRasterImages({
+	 exclude: /data-url\/.*$/,
+})],
+
+// Set a higher limit for appropriate directory
+[withRasterImages({
+	include: /data-url\/.*$/,
+	options: {
+		limit: 300000,
+	},
+})],
+```
+
+Using all three plugins with options acommodated to an example project structure:
+
+```js
+[withRasterImages({
+    exclude: /favicons\/.*$/,
+})],
+[withPlayback()],
+[withSVG({
+    exclude: [/images\/.*.svg$/, /favicons\/.*.svg/],
+    inline: true,
+})],
+[withSVG({
+    include: /images\/.*.svg$/,
+})],
+```
+
+
+## API
+
+
+### raster-images
+
+This plugin is meant to handle **rasterized images**, and tests the file types `.png`, `.jpg`, `jpeg`, `.gif`, `.webp` and`.ico`.
+
+
+### playback
+
+This plugin is meant to handle **video** and **audio** files, and tests the file types `.mp3`, `.flac`, `.wav`, `.aac`, `.ogg`, `.oga`, `.mp4`,  `.m4a`, `.webm` and `.ogv`.
+
+
+### svg 
+
+This plugin is meant to handle **SVG** files,  and tests the file type `.svg`.
+
+Though it defaults to working like the previous plugins, this plugin can also output inline content. You can toggle the output by sending an `inline` option, which is set to *false* by default. Use this if you want to be returned a `string` with the content of the SVG file. When *false* the plugin will behave like the others, using `url-loader`. When *true*, the plugin will use a different set of loaders, namely, and in order of execution,  [`svg-css-modules-loader`](https://github.com/kevin940726/svg-css-modules-loader), [`svgo-loader`](https://github.com/rpominov/svgo-loader) and [`raw-loader`](https://github.com/webpack-contrib/raw-loader).
+
+The available options also change in accordance with the `inline` value. With the `inline` option set to *false*, it will behave like the other plugins. With the `inline` option set to *true*, the options object will safely spread **only to the rule**, and passing the `use` option will **override the default loaders entirely**.
+
+```js
+// If false or not sent, options can be sent like other plugins
+[withSVG({
+    options: {
+        limit: 20000,    // will be safely passed to url-loader
+    },
+})],
+
+// If sent true, 'use' value will override default loaders entirely
+[withSVG({
+    exclude: /inline\/.*.svg$/,    // will be safely passed to rule
+    use: [{
+        loader: 'url-loader',     // only 'url-loader' will be used
+    }],
+})],
+```
+
+The following example shows how you can use the inline option in your project:
+```js
+// Inline SVGs are stored in /inline/
+[withSVG({
+	include: /inline\/.*.svg$/,
+	inline: true,
+})],
+
+// Exclude /inline/
+[withSVG({
+	exclude: /inline\/.*.svg$/,
+})],
+```
```

Comment:
We are missing two common sections: tests and license. See https://github.com/moxystudio/next-runtime-env#tests